### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by casz**
This is ready for review but I will resubmit the PR once we have the project transferred to Jenkins org

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,27 +8,32 @@
...
-        <version>3.47</version>
+        <version>3.50</version>
         <relativePath />
     </parent>
 
     <properties>
         <jenkins.version>2.176.1</jenkins.version>
         <java.level>8</java.level>
-        <workflow-step-api.version>2.20</workflow-step-api.version>
+        <configuration-as-code.version>1.30</configuration-as-code.version>
```
Comment (casz) [inline pom.xml]: Can be removed once https://github.com/jenkinsci/bom/pull/78 is included in the next release :)

**Comment by casz** [comment]
Based on https://issues.jenkins-ci.org/browse/HOSTING-832?focusedCommentId=375857&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-375857

I suggest we move forward with merging it :)

File: pom.xml [hunk 2]
```diff
@@ -8,31 +8,37 @@
...
     </properties>
 
     <groupId>io.jenkins.plugins</groupId>
-    <artifactId>dotcover</artifactId>
+    <artifactId>dotcoverrunner</artifactId>
```
Comment (rasmusjelsgaard) [inline pom.xml]: already done, rebase.

File: pom.xml [hunk 3]
```diff
@@ -187,9 +191,10 @@
     </build>
 
     <scm>
-        <developerConnection>scm:git@github.com:rasmusjelsgaard/dotcoverrunner-plugin.git</developerConnection>
-    <tag>HEAD</tag>
-  </scm>
+        <developerConnection>scm:git@github.com:jenkinsci/dotcoverrunner-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/dotcoverrunner-plugin</url>
```
Comment (rasmusjelsgaard) [inline pom.xml]: guess this should be done after the move is actually completed?

File: pom.xml [hunk 4]
```diff
@@ -8,31 +8,37 @@
...
         <jenkins.version>2.176.1</jenkins.version>
         <java.level>8</java.level>
-        <workflow-step-api.version>2.20</workflow-step-api.version>
+        <configuration-as-code.version>1.30</configuration-as-code.version>
     </properties>
 
     <groupId>io.jenkins.plugins</groupId>
-    <artifactId>dotcover</artifactId>
+    <artifactId>dotcoverrunner</artifactId>
+    <url>https://github.com/jenkinsci/dotcoverrunner-plugin</url>
```
Comment (rasmusjelsgaard) [inline pom.xml]: should be done after the move, right?

Comment (casz) [reply pom.xml]: Doesn't really matter if it is before or after :) Unless this is not the repo name we expect

File: pom.xml (hunk 3 above)
Comment (casz) [reply pom.xml]: Not like it matters, it will only be apart of the pom.xml and is only used for plugins.jenkins.io

**Pull request closed without merging**